src/web/components/ContentForm/RefWidget.js

```diff
@@ -1,12 +1,7 @@
 import React, { Component } from 'react'
-import { connect } from 'react-redux'
-
-import { fetchContentItem } from '~/actions'
 
 import ContentPickerWidget from '~/components/Content/Select/Widget'
 
-const decorateRef = id => `##ref(${id})`
```

Comment:
Why removing this method? I thought it's just nice to have it for semantics

Reply:
OK, I can leave it.